silk/model_factory.py

```diff
@@ -66,19 +66,20 @@ def encoded_headers(self):
         """
         From Django docs (https://docs.djangoproject.com/en/2.0/ref/request-response/#httprequest-objects):
         """
-        headers = {}
-        sensitive_headers = {'authorization'}
+        sensitive_headers = SilkyConfig().SILKY_SENSITIVE_KEYS
```

Comment:
maybe it's worth using 
```
set(SilkyConfig().SILKY_SENSITIVE_KEYS)
```
just for right type guarantee

Reply:
you're right
done